### ISSUE TITLE
tekton-lint complaining about TriggerBinding and EventListener with same name

### ISSUE THREAD

**Issue by jauninb** [IBM/tekton-lint#28]
tekton-lint (5.0) is complaining on the following file: https://github.com/open-toolchain/tekton-catalog/blob/master/git/sample-skip-ci/listener-skip-ci.yaml

```
git/sample-skip-ci/listener-skip-ci.yaml:
error   (54,9,54,22): 'github-commit' is already defined (as a 'TriggerBinding')
error   (69,9,69,22): 'github-commit' is already defined (as a 'EventListener')
...
```

AFAIK nothing prevent to use same name for different K8S resources/Tekton resources as long as they are of different types

**Comment by jauninb** [comment]
The tekton-lint was used for the whole https://github.com/open-toolchain/tekton-catalog and the linter seems to be valid according to the set: `$ tekton-lint --color --max-warnings 0 '**/*.yaml'`
```
git/sample-skip-ci/listener-skip-ci.yaml
git/sample-set-commit-status/listener-commit-status.yaml
```
The files above both have `github-commit` as TriggerBinding  and EventListener
